examples/mnist_tfrecord.py

```diff
@@ -0,0 +1,108 @@
+'''MNIST dataset with TensorFlow TFRecords.
```

Comment:
Please add a slightly more detailed explanation here: what are data tensors / TFRecords, why one may want to use them, how this workflow differs from the classic Keras workflow.